### DISCUSSION TITLE
Label customization in filter

### DISCUSSION THREAD

**Discussion by andresvennera** [Power-Components/livewire-powergrid#598] [Q&A] [answered]
Dears:
   I am starting to implement powergrid in a project. It works really very well. 
   What I can't do is something relatively simple: change the label of the column filters (makeInputText), for example I want to customize that instead of saying 'contains' it says "INCLUYE", etc. I have searched in the documentation and discussions and I can't find it. My idea is to use the same standard filter methods already included("Is/Is not, Contains/Does not contain, Starts with/Ends with...:"), and just change the label of them.
Thank you very much for the answer!
Regards

**Comment by dansysanalyst** [comment, marked as answer]
Hi @andresvennera,

PowerGrid ships with Spanish translations. See this issue answer explaining how to translate: https://github.com/Power-Components/livewire-powergrid/issues/190

The default translation is "_contiene_" and not "_incluye_", but you can customize the language files once they are published.

**Comment by andresvennera** [comment]
Hi Dan! Thanks a lot for your quickly response!!
